resources/sphinx/sphinx/conf.py.in

```diff
@@ -0,0 +1,271 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath('.')) # in order to use the local modules
+sys.path.insert(0, os.path.abspath("@PROJECT_SOURCE_DIR@/python"))
+import textwrap
+
+# cmake parser custom module
+from docutils.parsers.rst import Directive
+from docutils import nodes
+from sphinx.util.nodes import set_source_info
+currentmode = "user"
+
+# AutoStructify for math in markdown
+import recommonmark 
+from recommonmark.transform import AutoStructify
+
+# on_rtd is whether we are on readthedocs.org, this line of code grabbed from docs.readthedocs.org
+on_rtd = os.environ.get('READTHEDOCS', None) == 'True'
+
+
+# -- Project information -----------------------------------------------------
+
+project = u'@PROJECT_NAME@'
+copyright = (u'Copyright (c) 2020, New York University and ' +
+             u'Max Planck Gesellschaft. License under the BSD 3-clause.')
+author = u'Maximilien Naveau'
+
+# The short X.Y version
+version = u'1.0'
+# The full version, including alpha/beta/rc tags
+release = u'1.0.0'
+
+
+# -- General configuration ---------------------------------------------------
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    'sphinx.ext.autodoc',
+    'sphinx.ext.doctest',
+    'sphinx.ext.intersphinx',
+    'sphinx.ext.todo',
+    'sphinx.ext.coverage',
+    'sphinx.ext.mathjax', # doxygen math in doc
+    'sphinx.ext.ifconfig',
+    'sphinx.ext.viewcode',
+    'sphinx.ext.githubpages',
+    'cmake',
+    'recommonmark', # markdown support
+    'breathe', # to define the C++ api with breathe-apidoc
+]
+
+# Root of the github page:
+github_doc_root = 'https://machines-in-motion.github.io/code_documentation/@PROJECT_NAME@'
```

Comment:
This will probably also be problematic when used in repositories that don't belong to this github organisation?  But I could imagine it's not so easy to find a generic solution here?

Reply:
I am not sure if this does anything^^ I will try